deploy/docs/V1_Migration_doc.md

```diff
@@ -0,0 +1,177 @@
+| sumologic.fluentd.buffer 	| fluentd.buffer.type 	|
+| sumologic.fluentd.autoscaling.* 	| fluentd.logs.autoscaling.* , fluentd.metrics.autoscaling.* 	|
+| sumologic.k8sMetadataFilter.watch 	| fluentd.logs.containers.k8sMetadataFilter.watch 	|
+| sumologic.k8sMetadataFilter.verifySsl 	| fluentd.logs.containers.k8sMetadataFilter.verifySsl 	|
+| sumologic.k8sMetadataFilter.cacheSize 	| fluentd.metadata.cacheSize 	|
+| sumologic.k8sMetadataFilter.cacheTtl 	| fluentd.metadata.cacheTtl 	|
+| sumologic.k8sMetadataFilter.cacheRefresh 	| fluentd.metadata.cacheRefresh 	|
+| deployment.* 	| fluentd.statefulset.* 	|
+| eventsDeployment.* 	| fluentd.eventsStatefulset.* 	|
+
+
+   - `sumologic.kubernetesMeta` and `sumologic.kubernetesMetaReduce` have been removed. The default log format (`fluentd.logs.output.logFormat`) is `fields`, which removes the relevant metadata from the JSON body of the logs, making these configs no longer necessary.
+   - `sumologic.addStream` and `sumologic.addTime` (default values were `true`) have been removed; the default behavior will remain the same. To preserve the behavior of `addStream = false` or `addTime = false`, you can add the following config to the `values.yaml` file:
+
+```yaml
+fluentd:
+  logs:
+    containers:
+      extraFilterPluginConf:
+        <filter **>
+          @type record_modifier
+          remove_keys stream, time
+        </filter>
+```
+
+Until now, Helm users have not been able to modify their Fluentd configuration outside of the specific parameters that we exposed in the `values.yaml` file. Now, we expose the ability to modify the Fluentd configuration as needed. 
+
+Some use-cases include : 
+ - custom log pipelines, 
+ - adding Fluentd filter plugins (ex: fluentd throttle plugin), or 
+ - adding Fluentd output plugins (ex: forward to both Sumo and S3)
+
+The Fluentd `deployments` have been changed to `statefulsets` to support the use of persistent volumes. This will allow better buffering behavior. They also now include `“fluentd”` in their names. This is not a breaking change for Helm users.
+
+The unified Fluentd `statefulsets` have been split into set of two different Fluentd's, one for `logs` and the other one for `metrics`.
+
+### How to upgrade
+**Note: The below steps are using Helm 2. Helm 3 is not supported.**
+#### 1. Upgrade to helm chart version `v0.17.1 `
+
+Run the below command to fetch the latest helm chart:
+```bash
+helm repo update
+
+If the user is not already on `v0.17.1` of the helm chart, upgrade to that version first by running the below command.
+
+```bash
+helm upgrade collection sumologic/sumologic --reuse-values --version=0.17.1
+```
+#### 2: Run upgrade script
+
+For Helm users, the only breaking changes are the renamed config parameters.
+For users who use a `values.yaml` file, we will provide a script that customers can run to convert their existing `values.yaml` files into ones that are compatible with the major release. 
```

Comment:
nit: we can change `customers` to `users` to stay consistent with wording

Reply:
nit: `we provide` rather than `we will provide`
nit: singular for values.yaml files, so
```
... convert their existing `values.yaml` file into one that is compatible with ...
```